website/docs/usage/models.md

```diff
@@ -59,6 +38,31 @@ your data.
 > $ pip install -U %%SPACY_PKG_NAME[lookups]%%SPACY_PKG_FLAGS
 > ```
 
+If a trained pipeline is available for a language, you can download it using the
+[`spacy download`](/api/cli#download) command as shown above. In order to use
+languages that don't yet come with a trained pipeline, you have to import them
+directly, or use [`spacy.blank`](/api/top-level#spacy.blank):
+
+```python
+from spacy.lang.fi import Finnish
+nlp = Finnish()  # use directly
+nlp = spacy.blank("fi")  # blank instance
+```
+
+Initializing the language object directly yields the same result as generating
+it using `spacy.blank()`. In both cases the default configuration for the chosen
+language is loaded, and no pretrained components will be available.
```

Comment:
I would explain what a blank pipeline actually does and why you'd want it. Something like:

> A blank pipeline is typically just a tokenizer. You could create a blank pipeline when you only need a tokenizer, when you want to add more components from scratch, or for testing purposes.